RandomTips.js

```diff
@@ -322,15 +322,15 @@ export async function showRandomTip() {
  * Shows the random tip only randomly so the user is not annoyed.
  *
  * @public
- * @returns {Promise}
+ * @returns {Promise | undefined}
```

Comment:
I believe the TypeScript compiler would complain about this...
```suggestion
 * @returns {Promise<void> | void}
```